pages/runtime/runtime.md

```diff
@@ -1,21 +1,83 @@
 ---
-title: Runtime Environment
+title: Container Runtime
 ---
+# Container Runtime
+
+On resin.io devices all your application code runs a [ Docker container][container-link]. This means that whatever you define as `CMD` in your `Dockerfile` will be PID 1 of the process tree in your container. It also means that this PID 1 process needs to know how to properly process UNIX signals, reap orphan zombie processes [[1]][ref-one] and if it crashes your whole container crashes, losing logs and debug info.
+
+## Init System
+
+For these reasons we have built an [init system][init-system-link] into most of the resin base images listed here: [Resin Base Images Wiki][base-image-wiki-link]. The init system will handle signals, reap zombies and also properly handle [udev][udev-link] hardware events correctly.
+
+There are two ways of enabling the init system in your application. You can either add the following environment variable in your Dockerfile:
+```Dockerfile
+# enable container init system.
+ENV INITSYSTEM=on
+```
+or you can add an environment variable from the Dashboard by navigating to the `Environment Variables` menu item on the left and adding the variable as show below:
+![Enable init system](/img/common/app/app_initsystem_envvar.png)
+
+Once you have enabled your init system you should see something like this in your device logs:
+![init system enabled in logs](/img/common/device/device_logs_initsystem_enabled.png)
+
+You shouldn't need to make any adjustments to your code or `CMD` it should just work out of the box. Note that if you are using our Debian or Fedora based images, then you should have [systemd][systemd-link] in your containers, where as if you use one of our Alpine images you will have [OpenRC][openrc-link] as your init system.
```

Comment:
You shouldn't need to make any adjustments to your code or `CMD`. Why not say you **won't** need to make any adjustments? Are there cases when the init system messes with someones app?
 
where as if you use -> whereas if you use